src/amp-story-player.js

```diff
@@ -44,6 +44,17 @@ const IframePosition = {
   NEXT: 1,
 };
 
+/**
+ * @enum {number}
+ */
+const SwipingState = {
+  SWIPING_TO_NEXT: 0,
+  SWIPING_TO_PREVIOUS: 1,
+};
```

Comment:
Would `SWIPING_TO_LEFT/RIGHT` make it easier to solve for RTL support? The gesture recognizer would be tied to the content or its direction.